bin/did-calypso-app-change.mjs

```diff
@@ -28,18 +51,18 @@ async function downloadPrevBuild( appSlug, dir ) {
 	const prevBuildZip = `${ dir }/prev-archive-download.zip`;
 	const stream = createWriteStream( prevBuildZip );
 
-	const prevBuildUrl = `${ process.env.tc_sever_url }/repository/download/calypso_calypso_WPComPlugins_Build_Plugins/${ appSlug }-release-build.tcbuildtag/${ appSlug }.zip?guest=1&branch=try-parallel-app-processing`;
+	// TODO change branch to trunk eventually.
+	const prevBuildUrl = `${ process.env.tc_sever_url }/repository/download/calypso_calypso_WPComPlugins_Build_Plugins/${ appSlug }-release-build.tcbuildtag/${ appSlug }.zip?guest=1&branch=combined-calypso-apps`;
```

Comment:
I'll need to change the branch to trunk before merging.

Reply:
Leaving a note so we don't forget to change it before shipping.